spec/kafka_spec.rb

```diff
@@ -10,6 +10,8 @@
     ChefSpec::SoloRunner.new do |node|
       node.set["confluent"]["kafka"]["server.properties"]["key"] = "value1"
       node.set["confluent"]["kafka"]["log4j.properties"]["key"] = "log1"
+      node.set['confluent']['kafka']['server.properties']['broker.id'] = 1
+      node.set['confluent']['kafka']['zookeepers'] = 'testhost.chefspec'
     end
```

Comment:
the are in kitchen.  I can add them for chefspec as well.  

will push shortly.